### ISSUE TITLE
Option to hide the menu button - with space preserved - for Windows that are not maximized

### ISSUE THREAD

**Issue by maverick74** [psifidotos/applet-window-appmenu#161]
There should be an option to hide the menu button ( but preserving the space it needs when it's visible, as blank ) for when the window is not maximized!

It's basically the same thing we already have on the window control buttons plasmoid.

Here's the idea:

I have placed the application menu button (the one that comes standard with plasma) on the window titlebar (this automatically hides the menu bar from the application).

When the window is NOT maximized it has the titlebar shown (with the menu button and the window control buttons)

On the panel the window control buttons are hidden (but the space where they show is preserved as blank) - because, since they are already on the window frame it's redundant to have them on the panel as well.

I propose the same approach to the menu button.
in windows that are NOT maximized the menu button should be invisible - but have it's space preserved as a blank space. This because we already have the menu button on the window titlebar and it's redundant to have it on the panel as well.

When I maximize the window I have it set for the window to hide the titlebar/frame and it is at this point that the menu button should became visible on the panel.

**Comment by leuchtrakete666** [comment]
i completely agree. i thought its some kind of bug, that the burger menu don't disappear, when the window is not maximized, or even when there is no window at all. all windows closed and the burger menu is still there. grayed out, not responsive but still visible.

**Comment by maverick74** [comment]
@psifidotos Any chance for this soon? :)

Thank you
